scripts/openstack-quickstart-demosetup

```diff
@@ -264,7 +264,8 @@ EODASHDB
 SESSION_COOKIE_SECURE = CSRF_COOKIE_SECURE = USE_SSL
 EOSEC
     # sync dashboard DB "after" the database is created
-    run_as wwwrun "cd /srv/www/openstack-dashboard; umask 0027; python -m 'manage' syncdb --noinput"
+    run_as wwwrun "cd /srv/www/openstack-dashboard; umask 0027; python -m 'manage' makemigrations"
```

Comment:
Right, this is not needed / recommended unless the migrations are not distributed with the code. But if there are not migrations, this can be needed.

Also not sure why manage is quoted